model-optimizer/extensions/ops/elementwise.py

```diff
@@ -205,9 +206,17 @@ class Minimum(Elementwise):
 class Round(Elementwise):
     enabled = False
     op = 'Round'
-    op_type = None
-    version = 'extension'
+    op_type = 'Round'
+    version = 'opset4'
     operation = staticmethod(lambda a: np.round(a))
```

Comment:
The numpy `round` method (https://docs.scipy.org/doc/numpy-1.17.0/reference/generated/numpy.around.html#numpy.around) does not support `mode` with `half_away_from_zero` and you don't check it here. Please, implement an infer function which covers both modes.

Reply:
Done, now infer function supports both modes.